### ISSUE TITLE
PowerPC build fails

### ISSUE THREAD

**Issue by gaul** [s3fs-fuse/s3fs-fuse#875]
It is unclear to me whether this ever worked or regressed.  @Basavaraju013 can you investigate this?  References #866.

**Comment by gaul** [comment]
Should we revert the change or can you investigate this?

**Comment by Basavaraju013** [comment]
would you change this line from trusty to xenial.
https://github.com/s3fs-fuse/s3fs-fuse/blob/master/.travis.yml#L48

**Comment by gaul** [comment]
Can you submit a PR for this?

**Comment by Basavaraju013** [comment]
> Can you submit a PR for this?

Sure.

**Comment by Basavaraju013** [comment]
it issue with travis-ci.org. both amd64 and ppc64le jobs are running on amd64 machine only.
https://travis-ci.org/s3fs-fuse/s3fs-fuse/jobs/468559312#L495

Actually amd64 job need to run on amd64 and PowerPC job need to run on ppc64le VM. 
but in now both jobs are running on x86 only. Checking with travis-ci support team on this issue.
Here the documentation support of ppc64le machine on travis-ci.org
https://travis-ci.community/t/linux-multiarch-testing-tips/862

**Comment by Basavaraju013** [comment]
@gaul Travis-ci.org has fixed the RabitMQ authentication details now. 
Below is response from travis-ci support team.

```
Hi Basavaraju,

Thanks for your patience.

I talked with our engineering team and it appears there was a problem with IBM's RabbitMQ credentials.

This should be fixed now though, so can you please try running a build and let us know how it goes?

Look forward to hearing from you.

--
Luna G.

Travis CI Test and Deploy with Confidence.
www.travis-ci.com
```

and also builds are passing now (https://travis-ci.org/s3fs-fuse/s3fs-fuse/builds/470752015)
can we close this issue now.

**Comment by gaul** [comment]
This appears fixed (#874 will fix macOS failure):

https://travis-ci.org/s3fs-fuse/s3fs-fuse/builds/470774703
